e2e-playwright/plugin-e2e/plugin-e2e-api-tests/as-admin-user/panelEditPage.spec.ts

```diff
@@ -63,9 +69,9 @@ test.describe(
           formatExpectError('Did not expect panel error to be displayed after query execution')
         ).toBeHidden();
         await expect(
-          panelEditPage.getByGrafanaSelector(selectors.components.Panels.Visualization.Table.body),
+          page.locator('.rdg'),
```

Comment:
Using a css class as a locator is very brittle and would potentially break easily with future changes, consider adding a test-id to the expected element

Reply:
yep, thankfully I found an alternative yesterday that I'm gonna swap these to. The element we have to target here has changed in TableNG from one we author to one authored by a third party library. That class name isn't generated, so it's brittleness is less than you might expect (`.rdg` stands for [react-data-grid](https://github.com/adazzle/react-data-grid)), but we'll still just target `role="grid"` instead of the class name.